gbpservice/neutron/nfp/configurator/api/v1/controllers/__init__.py

```diff
@@ -0,0 +1,42 @@
+from wsmeext import pecan as wsme_pecan
+from wsme import types as wtypes
+
+import controller
+
+"""this class call methods of Controller class
+according to HTTP request"""
+
+
+class ControllerResolver(object):
+
+    create_network_function_device_config = controller.Controller(
```

Comment:
Why is this outside __init__() function. What is the difference between both the approaches?